<launch>

    <node name="canbus_driver" pkg="autolabor_canbus_driver" type="canbus_driver" output="screen">
        <param name="port_name" value="/dev/ttyUSB0" />
    </node>

    <node name="pm1_driver" pkg="autolabor_canbus_driver" type="pm1_driver" output="screen" />

</launch>
